dependency-manager-tests/spm/SPMProject/AppDelegate.swift

```diff
@@ -5,74 +5,19 @@
 */
 
 import UIKit
-import CoreData
```

Comment:
i can imagine removing this line made you happy 😄

Reply:
Totally 😄. I also noticed that the "+" button in navigation bar of the skeleton app was not working - it was crashing the app. Turns out that creating "basic Core Data" example requires some custom logic to be implemented.

Anyway, I changed all examples to "Single View App".